kotlin/samples/todo-android/common/src/main/java/com/squareup/sample/todo/TodoListsAppWorkflow.kt

```diff
@@ -59,22 +60,20 @@ object TodoListsAppWorkflow :
   private val listsWorkflow = TodoListsWorkflow()
   private val editorWorkflow = TodoEditorWorkflow()
 
-  private fun onListSelected(index: Int): TodoListsAction = WorkflowAction {
-    state = EditingList(state.lists, index)
-    null
```

Comment:
I really like this change, no need to explicitly returning `null` anymore in cases like this (most of the cases from my experience).